packages/web-runtime/src/App.vue

```diff
@@ -203,6 +203,7 @@ export default defineComponent({
 <style lang="scss">
 body {
   margin: 0;
+  font-size: 0.88rem;
```

Comment:
Please use css variables from the design system for font sizes

Reply:
yes will change it, was just an test